### ISSUE TITLE
(SOD) Add Missile Barrage

### ISSUE THREAD

**Issue by deadmeu** [ennvina/spellactivationoverlay#179]
Could you please add an overlay for Missile Barrage (Waist rune)?

It looks like there is some existing code but in-game there is no overlay displayed when it procs.

**Comment by ennvina** [comment]
Hi and thank you for your feedback.

## TL;DR

This will be included in the next release, with a slight blue-ish tint. Stay tuned!

## Long story

The code that you saw was designed for Wrath of the Lich King, where Missile Barrage is a talent in the Arcane tree.

During phase 1 of Season of Discovery, we started to support Arcane Blast, which uses the same texture as Wrath's Arcane Blast.

But because Season of Discovery now also has a Missile Barrage effect, a conflict is possible between Arcane Blast and Missile Barrage.

It was a honest mistake from our end, because we didn't know back then that both effect would end up in the same game content.

To mitigate this issue, the current decision is to make Missile Barrage blue-ish. We might iterate on this in the future. Feedback if, of course, appreciated. Thank you :)

**Comment by ennvina** [comment]
Hi, Missile Barrage has been added in #183 and #189 and shall be included in the next release.
